action.yml

```diff
@@ -205,6 +250,39 @@ runs:
       path: .tmp-ansible-collection-checkout
       persist-credentials: false
       ref: ${{ inputs.git-checkout-ref }}
+      fetch-depth: >-
+        ${{
+          steps.compute-change-detection-coverage.outputs.change-detection-arg
+          && '0' || '1'
+        }}
+
+  - name: Log the next step action
+    if: >-
+      !inputs.collection-src-directory
+      && steps.compute-change-detection-coverage.outputs.change-detection-arg
+    run: >-
+      echo ▷ Create branches for change detection
+    shell: bash
+  - name: Create branches for change detection
+    if: >-
```

Comment:
Does this need to also take into account whether it's a PR?

Reply:
It already does, since `steps.compute-change-detection-coverage.outputs.change-detection-arg` is an empty string if this is not a PR.